ludwig/features/category_feature.py

```diff
@@ -126,8 +125,9 @@ def get_feature_meta(column, preprocessing_parameters, backend):
         )
         vocab_size = len(str2idx)
         if vocab_size <= 1:
-            raise InputDataError(
-                column.name, CATEGORY, f"At least 2 distinct values are required, column only contains {str(idx2str)}"
+            logger.warning(
+                f"Feature {column.name} of type {CATEGORY} only contains {vocab_size} distinct "
+                "values: {str(idx2str)}. Consider dropping the feature."
```

Comment:
I think you may need to add an `f` before `"values: {str(idx2str)}. Consider dropping the feature."`, but if not, this looks good! Thanks for making the change!

Reply:
Good catch!